documents/entityrelationship/service.go

```diff
@@ -36,7 +36,7 @@ type Service interface {
 // service always returns errors of type `errors.Error` or `errors.TypedError`
 type service struct {
 	documents.Service
-	repo      documents.Repository
+	repo      repository
```

Comment:
shouldn't we add this in addition to the documents.Repo? otherwise we cannot access the documents.Repo, repo is just cast as a storage.Repo